Formula/i/idsgrep.rb

```diff
@@ -0,0 +1,37 @@
+class Idsgrep < Formula
+  desc "Grep for Extended Ideographic Description Sequences"
+  homepage "https://tsukurimashou.org/idsgrep.php.en"
+  url "https://tsukurimashou.org/files/idsgrep-0.6.tar.gz"
+  sha256 "2c07029bab12d9ceefddf447ce4213535b68d020b093a593190c2afa8a577c7c"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*idsgrep[._-]v?(\d+(?:\.\d+)+)\.t*/i)
+  end
+
+  depends_on "cmake" => :build
+
+  def install
+    system "./configure", "--disable-silent-rules", "--with-pcre=no"
```

Comment:
on a second thought, maybe let's pcre support back, thanks! 🙏 

did you find where the code is hosted? maybe we can also file an issue to track it.